ComputerVisionServer/moedx/utilities/utilization.py

```diff
@@ -213,17 +137,15 @@ def usage_cpu_and_mem():
     if stats_mem_util.n > 0:
         logger.info("====> Average Memory Utilization=%.1f%%" %(stats_mem_util.mean()))
     if stats_gpu_util.n > 0:
-        logger.info("====> Average GPU Utilization Snapshot=%.1f%%" %(stats_gpu_util.mean()))
-    if stats_gpu_util_max.n > 0:
-        logger.info("====> Average GPU Utilization Max=%.1f%%" %(stats_gpu_util_max.mean()))
-    if stats_gpu_util_avg > 0:
-        logger.info("====> Average GPU Utilization Avg=%.1f%%" %(stats_gpu_util_avg))
+        logger.info("====> Average GPU Utilization=%.1f%%" %(stats_gpu_util.mean()))
     if stats_gpu_mem_util.n > 0:
-        logger.info("====> Average GPU Memory Utilization Snapshot=%.1f%%" %(stats_gpu_mem_util.mean()))
-    if stats_gpu_mem_util_max.n > 0:
-        logger.info("====> Average GPU Memory Utilization Max=%.1f%%" %(stats_gpu_mem_util_max.mean()))
-    if stats_gpu_mem_util_avg > 0:
-        logger.info("====> Average GPU Memory Utilization Avg=%.1f%%" %(stats_gpu_mem_util_avg))
-    print("%.1f, %.1f, %.1f, %.1f, %.1f, %.1f, %.1f, %.1f" %(stats_cpu_util.mean(), stats_mem_util.mean(),
-        stats_gpu_util.mean(), stats_gpu_util_max.mean(), stats_gpu_util_avg,
-        stats_gpu_mem_util.mean(), stats_gpu_mem_util_max.mean(), stats_gpu_mem_util_avg))
+        logger.info("====> Average GPU Memory Utilization=%.1f%%" %(stats_gpu_mem_util.mean()))
+    if stats_memory_total.n > 0:
+        logger.info("====> Average GPU Memory Total=%d" %(stats_memory_total.mean()))
```

Comment:
Just a question why the average of the total memory, Why not just the total memory?

Reply:
Or you mean total memory used?

Reply:
That's a good point. I don't believe the total memory can be changed during runtime, so the average will always be the same value and every sample of the total will be the same value. Just force of habit that I did it that way.